src/lsp/cobol_ast/abstract.ml

```diff
@@ -13,39 +13,54 @@
 
 module type MISC_SECTIONS = sig
   (* IDENTIFICATION_DIVISION *)
-  type informational_paragraphs [@@deriving show]
-  type options_paragraph        [@@deriving show]
-  type environment_division     [@@deriving show]
+  type[@deriving ord] informational_paragraphs
```

Comment:
I'm not a huge fan of putting attributes/extensions here as I often search with regexps using strings like "`type <typename-prefix>`" when looking for type definitions sometimes (we all have our old-school ways of editing). I'm also wondering, wasn't `[@deriving show, ord]` enough to declare the `pp_` functions?

Reply:
`[@@deriving show]` declares the `pp_` functions but also `show_` functions that are never used. I can move the deriving to paragraph attributes.

Reply:
Well, we derive `show_` for most of the other types; we can keep those in favor of concision of the source code.

Reply:
Ah but you did not implement the `show_` functions indeed.

Reply:
I mean I can implement `show_` but figured that since it is not used it was not necessary :) I can change it to use `deriving show` and implement `show_` if you want.

Reply:
Well… I guess that makes a lot of single-line definitions to add. I'd like to quickly merge some of my changes as well, but these touch the AST so we can ignore `show`'s for now.